addons/l10n_rs_edi/models/account_move.py

```diff
@@ -0,0 +1,157 @@
+
+    l10n_rs_edi_is_eligible = fields.Boolean(
+        compute="_compute_l10n_rs_edi_is_eligible",
+        help="Serbia: technical field to determine if this invoice is eligible to be e-invoiced.",
+    )
+
+    l10n_rs_edi_attachment_file = fields.Binary(
+        string="Serbian E-Invoice XML File",
+        copy=False,
+        attachment=True,
+        help="Serbia: technical field holding the e-invoice XML data.",
+    )
+
+    l10n_rs_edi_attachment_id = fields.Many2one(
+        comodel_name='ir.attachment',
+        string="eFaktura XML Attachment",
+        compute=lambda self: self._compute_linked_attachment_id('l10n_rs_edi_attachment_id', 'l10n_rs_edi_attachment_file'),
+        depends=['l10n_rs_edi_attachment_file'],
+    )
+
+    l10n_rs_edi_state = fields.Selection(
+        string="Serbia E-Invoice state",
+        selection=[
+            ('to_send', 'Sending'),
+            ('sent', 'sent'),
+            ('sending_failed', 'Error')
+        ],
+        tracking=True,
+        readonly=True,
+        copy=False,
+    )
+
+    l10n_rs_edi_error = fields.Text(
+        string="Serbia E-Invoice error",
+        copy=False,
+        readonly=True,
+    )
+
+    l10n_rs_tax_date_obligations_code = fields.Selection(
+        string="Tax Date Obligations",
+        selection=[
+            ('35', 'By Delivery Date'),
+            ('3', 'By Issuance Date'),
+            ('432', 'By Billing System'),
+        ],
+        default='3',
+    )
+    l10n_rs_edi_invoice_id = fields.Char("Invoice Id", copy=False)
+    l10n_rs_edi_sales_invoice_id = fields.Char("Sales Invoice Id", copy=False)
+    l10n_rs_edi_purchase_invoice_id = fields.Char("Purchase Invoice Id", copy=False)
+
+    @api.depends('country_code', 'move_type')
+    def _compute_show_delivery_date(self):
+        # EXTENDS 'account'
+        super()._compute_show_delivery_date()
+        for move in self:
+            if move.country_code == 'RS':
+                move.show_delivery_date = move.is_sale_document()
+
+    @api.depends("country_code", "move_type")
+    def _compute_l10n_rs_edi_is_eligible(self):
+        for move in self:
+            move.l10n_rs_edi_is_eligible = move.country_code == 'RS' and move.is_sale_document()
+
+    def _post(self, soft=True):
+        posted = super()._post(soft)
+        posted.l10n_rs_edi_state = 'to_send'
+        return posted
+
+    @api.depends('l10n_rs_edi_state')
+    def _compute_show_reset_to_draft_button(self):
+        # EXTENDS 'account'
+        super()._compute_show_reset_to_draft_button()
+        for move in self:
+            move.show_reset_to_draft_button = move.l10n_rs_edi_state != 'sent' and move.show_reset_to_draft_button
+
+    def button_draft(self):
+        # EXTENDS 'account'
+        self.write(
+            {
+                "l10n_rs_edi_error": False,
+                "l10n_rs_edi_attachment_file": False,
+                "l10n_rs_edi_attachment_id": False,
```

Comment:
I hope those attachments are still accessible somewhere.
Also, `l10n_rs_edi_attachment_id` is computed so writing `False` has no effect

Reply:
Thinking about it again, setting the rs attachment to False doesn't make sense as the button is hidden when the invoice is sent. 
setting the error message and states to False instead